docs/4.operation/5.rancher.md

```diff
@@ -0,0 +1,22 @@
+## Rancher
+
+FIWARE Ops team is currently deploying a customised version of
+[Rancher 1.6](https://rancher.com/docs/rancher/v1.6/en/) on top of FIWARE Lab.
+This tool allows users with a Cloud account in FIWARE Lab to deploy and
+manage a Docker Swarm cluster on FIWARE Lab. Through which they can
+deploy their FIWARE applications on the Lab using docker containers.
+This is a service offered to the users, and as such FIWARE Lab Node
+Administrators do not have much to do. They only need to be sure that:
+
+-   Their OpenStack APIs are reachable from FIWARE Lab servers (which
+    should be the default case).
+
+-   Provide the configuration of MTU within their node to users
+    interested into using the service (or even installing manually
+    Docker Swarm).
+
+For more details, we recommend you to refer to the documentation of the
+Rancher set-up within FIWARE Lab:
+[*https://github.com/SmartInfrastructures/rancher-setup-notes*](https://github.com/SmartInfrastructures/rancher-setup-notes)
```

Comment:
page not found...

Reply:
Why did you delete this section? if there is a content about Rancher it makes sense to keep this paragraph to understand how to get details about use of Rancher in FIWARE Lab.

Reply:
That repository doesn't seem to exist anymore (or it was made private) and I couldn't find if it was moved somewhere, so I took the link out. We could open an issue about this if that resource isn't available right now.

Reply:
Ok, open an issue about this issue.

Reply:
Done, issue #15

Reply:
LGTM